scripts/Generate_ScopedFragmentTransformer.linq

```diff
@@ -19,13 +19,17 @@ void Main() {
     sb.AppendLine("    public HashSet<TSqlFragment> SkipList { get; } = new HashSet<TSqlFragment>();");
     sb.AppendLine("    ///<summary>Actions to perform when leaving a node.</summary>");
     sb.AppendLine("    public Dictionary<TSqlFragment, Queue<Action<TSqlFragment>>> PendingOnLeaveActionsByFragment { get; set; } = new Dictionary<TSqlFragment, Queue<Action<TSqlFragment>>>();");
+    sb.AppendLine("    Dictionary<int, object> visitorByTypeTag = new Dictionary<int, object>();");

```

Comment:
Since the tags are `ushort`, was there any benefit to making the dictionary a `ushort, object`?

Reply:
Oh yea - I added that to the script, but forgot to save it. I'll commit that now. Didn't notice a difference in the allocated bytes though.